pontoon/sync/files.py

```diff
@@ -0,0 +1,15 @@
+"""
+Various functions related to the paths and the io related stuff.
+"""
+
+import os
+
+
+def is_hidden(path):
+    """
+    Return true if path contains hidden directory.
+    """
+    for p in path.split(os.sep):
+        if p.startswith('.'):
+            return True
+    return False
```

Comment:
Actually, why don't we keep this inside `utils.py`?

Are there issues with import?